### PR TITLE
added global context and user cmpnt

### PR DESCRIPTION
**Pull request by tehtipsy**
- global context
 - user cmpnt

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,6 +1,11 @@
-import '../styles/globals.css'
-import type { AppProps } from 'next/app'
+import { GlobalContextProvider } from "@/contexts/globalContext";
+import "../styles/globals.css";
```
Comment (Kleinzit) [inline pages/_app.tsx]: "@/styles/globals.css"

File: components/base/myUser.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { GlobalContext } from "@/contexts/globalContext";
+import { useContext, useState } from "react";
+
+export const MyUserComponent = () => {
```
Comment (Kleinzit) [inline components/base/myUser.tsx]: Lets not call components "component"
also every component is "my" component
so how about "user form" or "user changer"?

File: pages/_app.tsx [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+import type { AppProps } from "next/app";
 
 export default function App({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  return (
+    <GlobalContextProvider>
+      <Component {...pageProps} />
+    </GlobalContextProvider>
```
Comment (Kleinzit) [inline pages/_app.tsx:9]: 🍆

File: pages/_app.tsx (hunk 1 above)
Comment (tehtipsy) [reply pages/_app.tsx]: yes

**Review by Kleinzit — APPROVED** [review]

**Pull request merged**